src/NHibernate/Mapping/OneToMany.cs

```diff
@@ -59,6 +59,8 @@ public IType Type
 		{
 			get { return EntityType; }
 		}
+		
+		public string PropertyName { get; set; }
```

Comment:
I think this new property should be removed as we don't need collection property name - we should just supply null to `EntityType` here